coral-component-table/src/scripts/Table.js

```diff
@@ -1598,6 +1598,10 @@ const Table = Decorator(class extends BaseComponent(HTMLTableElement) {
       // For icons (chevron up/down) styling
       headerCell[column.sortable ? 'setAttribute' : 'removeAttribute']('sortable', '');
 
+      if(headerCell.hasAttribute('sortable')){
+        headerCell.firstChild.setAttribute('role', 'button');
```

Comment:
please use same changes as done for coralui3

Reply:
@Pareesh I modified so the changes are the same as for coralui3. Also the build seems to fail because of a test that`s not related to my change "#**coral-columnview:navigate
        ✖ should be triggered when a column is removed**"